src/yaws_config.erl

```diff
@@ -3620,8 +3621,10 @@ str2term(Str0) ->
 
 check_ciphers([], _) ->
     ok;
-check_ciphers([Spec|Specs], L) ->
-    case lists:member(Spec, L) of
+check_ciphers([{Kex, Cipher, Mac, Prf} = Spec|Specs], L) ->
```

Comment:
To address the backwards compatibility issue for configured ciphers, could we add a clause to "up-convert" older ciphers?

    check_ciphers([{Kex, Cipher, Mac}|Specs, L) ->
        check_ciphers([{Kex, Cipher, Mac, default_prf}|Specs], L);

Reply:
Yeah maybe so. I wasn't sure how to handle the different PRF,
but perhaps the older ciphers just have to be fine with the
default anyway?

Reply:
Yeah, maybe. I guess if we took this approach and it broke some app because `default_prf` was not legal for a particular cipher, maybe it wouldn't be such a bad thing since then the app maintainer would be forced to deal with the problem correctly by adding the right PRF.

Reply:
So a best effort upgrade then? Perhaps it should be mentioned in the docs...